js/service/accountservice.js

```diff
@@ -23,6 +23,10 @@ define(function(require) {
 
 	function createAccount(config) {
 		return new Promise(function(resolve, reject) {
+			setTimeout(function() {
```

Comment:
@jancborchardt if you remove this and the return statement, the setup should work again.